app/models/institution_builder.rb

```diff
@@ -49,13 +49,21 @@ def self.run_insertions(version_number)
   end
 
   def self.run(user)
-    version = Version.create(production: false, user: user)
+    error_msg = nil
 
-    default_timestamps_to_now
-    run_insertions(version.number)
-    drop_default_timestamps
+    begin
+      Institution.transaction do
+        version = Version.create!(production: false, user: user)
 
-    version
+        default_timestamps_to_now
+        run_insertions(version.number)
+        drop_default_timestamps
+      end
+    rescue StandardError => e
```

Comment:
Do we have to rescue from StandardError? this makes debugging very difficult. Lets at least make sure we're logging something when this happens. But if we can get a more specific error to rescue from it will be better.